src/txmempool.cpp

```diff
@@ -216,6 +207,69 @@ bool CTxMemPool::CalculateMemPoolAncestors(const CTxMemPoolEntry &entry, setEntr
     return true;
 }
 
+bool CTxMemPool::CalculateMemPoolAncestors(const Package& package,
```

Comment:
This function doesn't need to have a `setAncestors` out param, since the only place that it's called doesn't use it. In fact, you could rename this function to something like `CheckPackageLimits()` to better document what it's used for.

Reply:
Good point - done :)